src/Microsoft.DotNet.Watcher.Core/project.json

```diff
@@ -1,5 +1,12 @@
 {
   "version": "1.0.0-*",
+  "description": "Dotnet watch core features. Contains the core logic for dotnet watch.",
```

Comment:
How about: `Core logic for the dotnet-watch command line tool.` or something like that?